### PR TITLE
Adds support for Longview Client endpoints

### PR DESCRIPTION
**Pull request by Dorthu**
Adds full support for Longview Clients, including getting a single
client, getting a list of clients, creating a client and updating a
client.  Also includes new fixtures and tests.

### CODE REVIEW

File: linode/linode_client.py [hunk 1]
```diff
@@ -199,6 +199,39 @@ def get_apps(self, *filters):
...
+    def create_client(self, label=None):
+        """
+        Creates a new LongviewClient, optinoally with a given label.
+
+        :param label: The label for the new client.  If None, a default label based
+            on the new client's ID will be used.
+
+        :returns: A new LongviewClient
+
+        :raises ApIError: If a non-200 status code is returned
```
Comment (SzySteve) [inline linode/linode_client.py]: I dont see this being raised

File: linode/linode_client.py [hunk 2]
```diff
@@ -199,6 +199,39 @@ def get_apps(self, *filters):
...
+    def get_clients(self, *filters):
+        """
+        Requests and returns a paginated list of LongviewClients on your
+        account.
+        """
+        return self.client._get_and_filter(LongviewClient, *filters)
+
+    def create_client(self, label=None):
+        """
+        Creates a new LongviewClient, optinoally with a given label.
```
Comment (SzySteve) [inline linode/linode_client.py]: optionally

File: linode/linode_client.py (hunk 1 above)
Comment (Dorthu) [reply linode/linode_client.py]: That error is raise in [_api_call](https://github.com/linode/python-linode-api/blob/master/linode/linode_client.py#L442) if the resulting status code indicates an error occurred - by extension, anything the contacts the api can raise it.  Should it not be included here?

Comment (SzySteve) [reply linode/linode_client.py]: you good just didnt see that. LGTM

**Pull request merged**